code/modules/awaymissions/corpse.dm

```diff
@@ -317,7 +317,7 @@
 /obj/effect/landmark/corpse/skeleton
 	name = "skeletal remains"
 	mobname = "skeleton"
-	mob_species = "skeleton"
+	mob_species = /datum/species/skeleton //why doesnt this use the species' id?

```

Comment:
because if you typo a word nothing happens.
if you typo a typepath byond shits itself making it VERY obvious to you that you made a mistake.

there, question answered, remove the comment.